sdk/resourcemanager/azure-resourcemanager-test/src/main/java/com/azure/resourcemanager/test/ResourceManagerTestProxyTestBase.java

```diff
@@ -230,14 +230,32 @@ protected String clientIdFromFile() {
     protected AzureUser azureCliSignedInUser() {
         AzureUser azureCliUser = new AzureUser(testResourceNamer);
         if (!isPlaybackMode()) {
-            String azCommand = "az ad signed-in-user show --output json";
+            // Due to AADSTS530084, Access to Graph API(scope) has been blocked by conditional access token protection policy configured by msft.
+            // We can't use "az ad" command anymore, nor auth to Graph scope. To learn more, see https://aka.ms/TBCADocs.
+            // Access to ARM scope token should be safe, as this is what we do in ARM auth.
+            String azCommand = "az account get-access-token --query accessToken -o tsv";
```

Comment:
Just try to understand more background": is `az ad` using 'AAD' in the background, and `az account` using 'Entra ID'?